### DISCUSSION TITLE
[Roadmap] Ideas for 2022 Q3 Sprint 1 Prioritisation

### DISCUSSION THREAD

**Discussion by BenJam** [opencollective/opencollective#5665] [Ideas]
This is the discussion to begin **surfacing ideas** for Q3S1 (25th July to 2nd September).

## Surfacing Ideas
While we've continued to iterate the process we use to drive (principally) development of the platform we have noticed that we often discuss ideas at face value as part of prioritisation. @iamronen [put it best in his assessment](https://www.loom.com/share/444c3753692e4f509739e165c4cfd95d):

> when it's time to eat we open the fridge and see what we can throw together, no one's thinking about and preparing a menu.

To counter this we would like to bring a little more attention to the process of readying ideas for prioritisation, Which starts with surfacing them here, in this discussion. 

## How to use this discussion
Surfacing an idea looks very similar to ideas that have been shared in previous discussions we ask that you think about _problems_, (not solutions) and the impact of those problems.  Consider the following questions:

- Who is your user?
- What are they trying to achieve?

## Strategic Priorities

- #5488
- #5487
- #5486 

It's useful to use the following format when trying to reflect this:

> When ____ , I want to ___ , so I can ___

We ask submitters to select problems that have both a high degree of impact and a high degree of alignment with our focus areas. If there's an existing issue, please reference it, if not please [click here to create one](https://github.com/opencollective/opencollective/issues/new?assignees=&labels=&template=ideas--irritations-and-needs.md&title=).

We encourage everyone to surface ideas _as early as possible_ so that we can start working toward a project _pitch_ before the next prioritisation week (July 18th). 

During the sprint (in parallel with development) we work together to create a pitch that can then be reviewed in prioritisation.

If you'd like to read more about pitches, or generally find out more about our preferred approach to development check out [Shape Up by Ryan Singer](https://basecamp.com/shapeup) (it's free)

## Another process iteration? What's the goal?

The goals for this change are: a greater degree of shared understanding, less pressure on sprint facilitators to understand every aspect of a problem in order to inform the prioritisation process and a smoother development process as engineers have more context at the outset.

**Comment by Memo-Es** [comment]
## Onboarding flow - Find a host #5672

**Stage:** Revisiting the shaped solution and writing a pitch.

**Comment by alanna** [comment]
## Stop users getting blocked in the expense flow
Due to problems with error validation and the form, users can get stuck in unresolvable states of the flow. This creates a broken experience for them so I think we should fix it.

* #5020
* #5306 
* #5429 
* #5222

**Reply by Memo-Es** [reply]
+ 1 to prioritize this for the next sprint, it is contained enough and a great improvement for the invite expense flow 

**Stage:** Ready to be pitched

**Reply by iamronen** [reply]
created issue and added to product board as P2 ready for pitching #5681

**Reply by Memo-Es** [reply]
Adding #5697 here to include it in this set of expense improvements

**Comment by alanna** [comment]
## #5230 Trigger Collective onboarding emails for new admins
We have an onboarding email series for creators of new Collectives, but don't do anything to give the same info to new admins of existing Collectives - it would be super helpful for their onboarding.

**Reply by iamronen** [reply]
@BenJam please have a look at this as it relates to notifications and may relate to the minimum number of admins change.

@alanna : Can you assign this a priority?

> counting days since they accepted the admin invite instead of days since the Collective was created.

This part of the requirement feels like a potential complication (depending on the architecture of the timing mechanism). 
One possible way to avoid this complication (while still delivering the value to the new admin) is to create a new notification for "new admin confirms invitation" and include in it the content from both messages in one message (note to notification authors: if in the future you change the notification sent to the first admin you also need to update the message sent to additional admins)

**Reply by alanna** [reply]
I have never understood how to assign priority really, sorry. This affects certain Collectives a lot (e.g. pretty much all Gift Collective admins completely miss out on the onboarding series, because I'm the one who creates the Collectives, plus all Collectives that go through an admin transition to someone not familiar with our platform), and this meaningfully impacts those Collective's outcomes and engagement. But it's a minority of Collectives that are in that situation. So wherever that fits into the prioritisation rubric.

**Reply by BenJam** [reply]
I've taken a look. @Memo-Es are you considering this as part of the #4829 project?

**Reply by Memo-Es** [reply]
As I posted inside the issue:
> I agree with @BenJam. We might want to create a new series of onboarding emails specifically for new admins of an existing Collective; the work then is to craft the content of these emails and add the respective templates in the database.

**Reply by BenJam** [reply]
@Memo-Es my question is: is this on your radar and should it not be prioritised because you've already set the next priority (find a host) for the onboarding work?

**Comment by alanna** [comment]
## #2734 Enable recurring bank transfer contributions
This one continues to get requested, and as the NZ hosts scale it will get more important (everything is done by bank transfer here). This also causes confusion when a host doesn't have stripe/paypal set up, and people click "contribute" then change to "recurring" and get hit with "no payment methods available" (see #4669).

**Reply by Memo-Es** [reply]
**Stage:** Shaping

**Reply by iamronen** [reply]
@BenJam I think this needs your attention.

**Reply by BenJam** [reply]
We've discussed this previously and, as @Betree has highlighted, there is [a workaround using PayPal](https://github.com/opencollective/opencollective/issues/2734#issuecomment-1099277212). 

Implementing this 'properly' requires a significant investment to remove some of the assumptions in our model and to perhaps [support SEPA](https://github.com/opencollective/opencollective/issues/1742) and ACH payments. @kewitz took a look at SEPA [back in 2019](https://github.com/opencollective/opencollective/issues/1742#issuecomment-466997000) and I don't think anything has changed. Stripe have reasonable support for this but it's a significant project. 

As a result my question back to you @alanna is, how big is the market for this solution? Specifically the NZ projects and [large host you mention here](https://github.com/opencollective/opencollective/issues/2734#issuecomment-949143843)? Last year collectives hosted by OCNZ received ~$65,000 for which it collects a 5% fee. To put that into perspective we'll probably spend ~$15-20k implementing this feature and I do think it's important to at least bear that in mind.

**Reply by kewitz** [reply]
To be honest, I'm more interested in working with digital wallets like Apple Pay and Google Pay because I think they're easy to implement and they address global payment problems.

SEPA will only address European payments which can be justified by OCEU and a huge potential market in Europe overall.
Also, people in NL are requesting support for iDEAL payments, which is yet another localized market payment method. https://github.com/opencollective/opencollective/issues/5618#issuecomment-1163067671

**Reply by natehn** [reply]
Noting here that there [was also some discussion](https://github.com/opencollective/opencollective/discussions/5511#discussioncomment-2700925) when this was submitted last sprint.

**Reply by znarf** [reply]
> SEPA will only address European payments which can be justified by OCEU

No, most Hosts can accept SEPA if we use the Stripe integration. This is huge for Business and Individuals willing to contribute from their european bank account.   

https://stripe.com/docs/payments/sepa-debit

<img width="865" alt="Screenshot 2022-06-30 at 11 18 20" src="https://user-images.githubusercontent.com/806/176641033-51cf781b-5078-4816-aaf9-005aa12baaa3.png">

**Comment by alanna** [comment]
## #3433  Upload attachments to expense comments
There's a large new host starting up in Australia who needs this, and it's been requested many times by users. I think it would simplify the process because the alternative is something like creating a google drive and linking to it from an expense comment but that's quite tricky when not all expense submitters have access to the google drive, etc etc. For Hosts, it's very helpful to keep all related documentation in one place: the expense.

**Reply by Memo-Es** [reply]
**Stage:** Ready to be pitched

**Reply by natehn** [reply]
I second this and previously [pitched it for Q2S2](https://github.com/opencollective/opencollective/discussions/5511#discussioncomment-2677025).

**Reply by Betree** [reply]
To clarify, this is about allowing PDFs in the rich text editors for expense comments. We already support attaching images.

**Comment by alanna** [comment]
## #5407 Legal name field when creating new user for added funds
Sometimes the desired display name is not the legal name of the donor, but Hosts must have the legal name recorded against the transaction. We can't rely on the contributor to go in an edit their legal name later when it comes to added funds transactions, so we should enable Host admins to enter it.

**Reply by BenJam** [reply]
asked a quick question as i wonder whether there's a harder req for c3 hosts to collect legal names on the platform.

**Reply by alanna** [reply]
Everything related to compliance is harder for tax deductible hosts (not just US c3s, we have charity hosts in NZ and Aus and other places too). But it's very good practice for all hosts to be able to record the names of contributors where possible for tax reporting purposes. Money laundering is actually a significant risk in our model and this is one tool for fighting it.

**Reply by Betree** [reply]
+1 for the consistency, that's a super simple change

**Comment by alanna** [comment]
## Invoice expenses must have payee address

Compliance requirement we are missing in some cases.
*  #5232
*  #5341

**Reply by Memo-Es** [reply]
I suggest bundling up this one together with the rest of the expenses maintenance issues https://github.com/opencollective/opencollective/discussions/5665#discussioncomment-2982019

**Comment by alanna** [comment]
## #5171 Update receipt when transaction is refunded
This one is especially important for tax-deductible hosts, as people should not be able to get documentation saying they are entitled to a tax deduction on money that was refunded.

**Reply by BenJam** [reply]
I'll take a look at this.

**Comment by alanna** [comment]
## More root actions
Bringing this one that @znarf suggested last time back in, because things like this save some of the most limited capacity we have, programmer and support hours.

* Ability to add/invite admins to a Collective
* Ability to move an Expense from a Collective to a Project/Event (and opposite)

**Reply by alanna** [reply]
@znarf could you create an issue for this?

**Comment by alanna** [comment]
## #5678 Improve cover image cropper

Suggested by @carolineawoolard last time. I keep running into this myself and it negatively impacts the experience of setting up a Collective and how it's presented to the world. I set up all the Gift Collective Collectives (due to how our onboarding process works) and I get caught on this every time.

**Reply by Memo-Es** [reply]
I would LOVE to work on the issues that Caroline describes ASAP. I'm considering this issue as part of the differentiation between profiles I describe [here](https://github.com/opencollective/opencollective/issues/4829#issuecomment-1148103264) (Part 1: Collective page revamp).
To be realistic, though, the next sprint does not sound doable to undertake a project that size since everyone is on vacation; I would leave it to the next one, @alanna. 

Note: I'm creating issues today to describe this issue better and mention it in this discussion.

**Reply by alanna** [reply]
Fair enough, we have to adjust expectations in line with capacity. I still think it's helpful to just collective everyone's wishlist here even if most of it won't be worked on soon.

**Reply by iamronen** [reply]
I understand the desire to "just collect everyone's wishlist" however we are trying to offer some kind of product-triage to help mature the wishes into more clearly scoped-pitches for prioritization. From that perspective this feels like flooding since unless stated otherwise (again: **from a product-triage perspective**) the "wishes" arrive here as all equally important.

Is that the case? are all the things on your wishlist equally important (to you, to the product)?

**Reply by alanna** [reply]
I don't know because 95% of my job is not about product and therefore I don't have space in my brain to load all the wider context, nor the time to do user interviews, gather data, etc. All I can say is, at least a few people have mentioned that this feels broken, and that displaying images well/poorly on profile pages makes an impact on the experience. To do a full assessment of relative priorities, we need someone whose job it is to hold that and can receive this kind of "flood" and process it however required to end up with a good result. If you put blockers on people sharing this kind if feedback, we will be missing all the voices of people whose main role is not product.

**Reply by alanna** [reply]
This is quite 'meta', but there is important context behind how this idea ended up on this list. This came from Caroline, who is not a programmer and hasn't felt like it was very accessible to put things forward into the sprint process until recently - but who brings a critically important perspective about comms (esp visual), engagement, UX, and accessibility. I would personally, for myself, probably never put an issue like this at the top of my list, because I'm not really a visual person. I wanted Caroline on the team specifically *because* she thinks differently to me, and sees different important, different things. This is representative of bigger issues around accessibility and diversity of inputs to prioritisation. We have to ensure there is a very accessible, low barrier way to drop in ideas and express needs. It great that you're thinking about deeper processes and more thoughtful traige, but realistically you can't outsource that back onto the people making suggestions, or you'll just miss out on input from everyone who doesn't have the time/inclination. We need to build up internal capacity on the team, to do that context building, triage, and analysis work. It has to be part of some one's, or some team's, job, because making it everyone's job isn't really feasible.

**Reply by Memo-Es** [reply]
I built [here](https://github.com/opencollective/opencollective/discussions/5665#discussioncomment-2990172) on the context of this issue and bundled it along with a couple of other issues related @alanna

**Reply by iamronen** [reply]
Thank you @alanna  for adding the "meta" context. I agree with you that allowing for more inclusive participation in shaping the platform is an important aspiration. For many years I sat on the sidelines of the open-source world with the confusion of not being able to offer input because I didn't speak the language or use the tools or have ideas that didn't fit into the existing paradigms. I would like to believe that extending the prioritization process from a few days to an entire cycle (and potentially longer) opens the door to more experimentation with how to do this.

> We need to build up internal capacity on the team

Yes, I believe this IS the current priority and we need to build this up as a well established internal capacity before inviting more input. As I have recently learned from you, these things need more time then we tend to give them :)

**Reply by iamronen** [reply]
some comments on the context of this issue:

1. The proposed title is misleading, because indeed it is about more than the image cropper.
2. The fact that some things are editable on the profile page and some are not is indeed confusing. I do believe it is better to be consistent about this, but I feel there are more important things to do about the profile page then allowing for inline-editing. I intend to speak more to this when we get to discussing the profile page redesign.
3.  The image-cropper can definitely use improvement, but before that I feel that the way cover images show up on profile pages needs to be reconsidered ... I hardly notice images on profile pages ... so much sure I am not even sure if/how often they are used. This issue will benefit from being considered within the context of the profile page redesign (and I believe @Memo-Es has bundled it there).
4. Caroline's request to add/edit team members resonates with an item on the OSC wish-list to make open-source project teams visible - not just admins and not just maintainers.

**Comment by alanna** [comment]
## #5395 Grant expenses should be opt-in by host
Currently grant expenses are on by default for Projects. This doesn't make much sense - why only Projects? If we are making a decision that they should default to on, they should be on for Collectives too. However, I strongly think grant expenses should be *off* by default. Hosts rely on our carefully designed invoice and receipt expense forms to capture required details for expense accounting, and grants go around all that (as a very generic kind of form). Hosts that use grants should think about if and how they want to receive such expenses, not get surprised when someone first submits one via a Project.

**Reply by iamronen** [reply]
ready for pitching added to product board,

**Comment by Memo-Es** [comment]
## https://github.com/opencollective/opencollective/issues/5680 Collective profile page revamp

Additional to https://github.com/opencollective/opencollective/discussions/5665#discussioncomment-2982082, this project is about scheduling some time to define this project further to have a clear path forward to improve it.

**Reply by BenJam** [reply]
@Memo-Es do we need to put some time into this to move to a pitch, or has it just been miscategorised?

**Reply by Memo-Es** [reply]
It is ready to be pitched the way I see it, will do it between today and tomorrow

**Reply by iamronen** [reply]
This is creating some confusion for me. 

To my understanding @Memo-Es your intention was to use this as a placeholder for shaping work to be done during the next cycle. 

If that is the case I am not sure this is necessary or the right place for it. In my mind that board is for things going into development. Things that we are working on shaping should only arrive here when they are ready to be pitched for developmemt.

If however, my understanding is incorrect, and you mean that the new profile page is already designed, then I would like to have a chance to look at the new designs before they are pitched.

**Reply by Memo-Es** [reply]
@iamronen I kinda want to stop using the "shaping," "ready to be pitched," etc. because no one but us understands them :)

The lifecycle of a project, the way I see it, is like this:

Identify > Definition phase > Design > Development > Repeat

With that structure in mind, this project is in the Definition + Design phase, I have the feeling that you are considering design work as part of the shaping phase, but in my mind, it has its phase before pushing to dev.

**Reply by iamronen** [reply]
@Memo-Es I look forward to discussing this further in our bi-weekly call.

1. I have added #5704 to this prioritization as a demonstration of what shaping can be. I think we have been misusing the term shaping and that needs to be corrected. I also agree that we will need to both become more clear and coherent about this within product & design and then communicate to the rest of the team about our discoveries and choices.
2. I think the actual lifecycle of a project is more complex then what you described. Your description does not account for a  multi-issue perspective - for example where does prioritization sit in the lifecycle? Prioritization is a process that looks at a set of projects as a whole. In order for that to happen, some work needs to take place before prioritization. Without this preparatory work prioritization is at best symbolic and mostly a rubber stamping process.  
3. An example of why this matters is what has been happening with Email Notification in Q2S2. I think that this issue was not clearly understood before it was prioritized and allocated engineering resources and as result has been more costly to develop than it could have been.  
4. For OSC a redesign of the profile page is an interesting and valuable project. I think there is more to it then is currently captured in #5680 which is why it does not feel to me ready to be prioritized. I would like to hear more about your thoughts and plans for the redesign, to collect my own thoughts on this and to share them with you so that we can shape a more comprehensive pitch. 
5. If you feel it is necessary, it is to separately prioritize some of the issues currently aggregated in #5680

**Reply by Memo-Es** [reply]
@iamronen, thank you for your take on this one; I feel excited to work on this project; I know it is a big issue!
 - Don't you think we can prioritize this project for the next sprint and have a round of shaping, idea sharing, and consensus as part of the shaping stage? I would say that's the right way to follow up with this project
 - Furthermore, during the prioritization round of conversations, we could prioritize 1 or 2 minor issues inside this issue that are ready for implementation (i.e., improving the cropping functionality https://github.com/opencollective/opencollective/issues/5678)

**Comment by kewitz** [comment]
### https://github.com/opencollective/opencollective/issues/5431 Feature: Manage pending Expenses and Charges

A new modal/view where users can see their pending expenses whether that is a recurring expense draft, a virtual card charge missing receipt or an expense marked as incomplete.

**Reply by iamronen** [reply]
@BenJam please take a look at this

to me this seems like a P4, maybe after release and adoption of recurring expenses it becomes a P3?

**Comment by kaylarep** [comment]
## https://github.com/opencollective/opencollective/issues/4556 Collective Admins want to Track Balance on Virtual Cards 

We get requests for this very frequently.

As a host admin, our collectives do not have an intuitive way of knowing how much they're spending (especially if multiple people are using the card.) 

They end up making as purchases as much as they can, eventually getting declined, reaching out for card balance increases (often times urgently because the payment needs to be by a certain deadline e.g. a storage unit, an Airbnb that requires a deposit within 24 hours), and we usually grant it. 

Some of our collectives are up to [$8000/month](https://opencollective.freshdesk.com/a/tickets/96896)  , because we don't want to restrict their usage (especially when they have an OC budget to cover); but this constant granting of increase doesn't feel scaleable or reliable.

**Reply by iamronen** [reply]
closed by @BenJam

**Reply by natehn** [reply]
I personally do not think this should be closed. Collectives would like to be able to see where they are in relation to the monthly limit.

**Comment by kaylarep** [comment]
## Papercut: https://github.com/opencollective/opencollective/issues/5689 Collective cards in the 'Contributions' section of their host: Appear as they do in search queries (with tags & budget)

As a fiscal host, this would help with amplification of our hosted groups. If people visit the hosts's OC page, they can easily see more detailed info about the hosted collectives.

**Reply by iamronen** [reply]
I suggest looking at this as part of #5680

**Comment by kaylarep** [comment]
## Papercut: https://github.com/opencollective/opencollective/issues/5692 Changing how Project virtual cards appear in fiscal host dashboard

This would be very helpful to fiscal host admins- especially if a single collective has many projects each with their own virtual cards.

**Reply by iamronen** [reply]
added as a pitch to product board

**Comment by natehn** [comment]
## Allow Host to turn off expense type(s) for Collectives https://github.com/opencollective/opencollective/issues/5525

I see this as a small change (but maybe I'm wrong and it isn't so small?) that can help make the platform more legible for certain kinds of fiscal hosts.

I previously [pitched this for Q2S2](https://github.com/opencollective/opencollective/discussions/5511#discussioncomment-2689805).

**Reply by natehn** [reply]
Perhaps relates to [Alanna's Grants papercut above](https://github.com/opencollective/opencollective/discussions/5665#discussioncomment-2982265)

**Reply by alanna** [reply]
Is this a dup of https://github.com/opencollective/opencollective/discussions/5665#discussioncomment-2982265

**Reply by natehn** [reply]
@alanna I do not think so - this is about more than just grants - one host wanted to turn off "invoice" expenses since they only do grants and reimbursements under their program

**Reply by iamronen** [reply]
@benjam please look at this

**Reply by natehn** [reply]
@poohlaga and I had a discussion about host expense types yesterday and I now think this is best folded into a larger discussion about types of expenses

**Comment by natehn** [comment]
## Papercut: Host admins can see Collective admins https://github.com/opencollective/opencollective/issues/5426

Per [Kayla](https://github.com/opencollective/opencollective/discussions/5511#discussioncomment-2677026):

> This is one is important, because sometimes collective admins want to hide their admins (which is fine by me), but at minimum, their fiscal host should be able to see that they have admins - especially if minimum # of admins are required by the host

**Reply by iamronen** [reply]
I would like to suggest holding off on this and addressing it systemically in the context of both #5680 and #5270 
@natehn and @kaylarep do you think this can wait?

**Reply by natehn** [reply]
Yes, I think so. It only comes up here and there

**Comment by kaylarep** [comment]
## https://github.com/opencollective/opencollective/issues/5695 Allow virtual card assignees to pause/freeze their own card

I can't think of any reasons why this wouldn't be ok. If anyone can, please share!

**Reply by iamronen** [reply]
@benjam I've added this to the product board as requires shaping (not ready for pitching)

**Reply by BenJam** [reply]
I think it's reasonably good to go as is.

**Comment by kaylarep** [comment]
## https://github.com/opencollective/opencollective/issues/5696 Notifications for virtual card purchases

As requested by a collective admin. This would provide helpful oversight for collective admins- especially since multiple people sometimes use the same card number.

**Reply by iamronen** [reply]
added to board by @BenJam

**Comment by alanna** [comment]
## Manually subscribe to Updates #1445

This one has potential big impact for our "communication" priority. It's been consistently requested since 2018 and it could be time. By way of evidence, Updates is a regularly used feature and it would be even more used if it could function more like a newsletter people could sign up for. We currently see users hacking a workaround using a free event to enable someone to become a 'contributor' without paying and thus subscribe. This indicates people want this feature, and if we properly supported it, it could add value. Other crowdfunding and donation management tools emphasise comms and engagement as a very important factor in fundraising success, and community engagement is one of OC's main value-adds as a product.

![Screen Shot 2022-06-28 at 9 28 06 AM](https://user-images.githubusercontent.com/5498878/176038687-ce5c0341-99a8-4db4-8e80-79e54fdd8bb8.png)

**Reply by natehn** [reply]
It was recently requested again [on Slack](https://opencollective.slack.com/archives/C0HSLRNVC/p1656350619565649).

**Reply by BenJam** [reply]
jeeeeeeeees that's a big thread. Seems like @Betree has done all the necessary work and that there's a reasonably simple approach there.

**Reply by Memo-Es** [reply]
I really like the challenge presented in this issue! I think it needs some design exploration to flesh out a coherent MVP and shape the other little things that complete the experience: Managing subscriptions, the concept of membership levels for communities (And stop calling everyone just contributors), the concept of having a way to interact with Collectives without the need of contributing financially (i.e., a 'like' or 'kudos') among other things Big +1!

**Status:** Need some shape-up work

**Reply by iamronen** [reply]
added to product board as requires shaping

**Comment by emberwry** [comment]
### #5033 create tick box for paypal fees to be paid by expensee (like we have for wise)

Right now, if someone is paying out most or all of their collective balance to a wise transfer, we (host admin) can tick "the payee is covering the fees" when there isn't quite enough to cover payment processing fees in the balance. We really need this option for paypal payments as well. This has come up a few different ways and looks like it's ongoing (since 2018!), and it definitely results in human error (redundant payments, clunky manually changing amounts and re-approving).

**Reply by znarf** [reply]
Check https://github.com/opencollective/opencollective/issues/5033 for latest update.

**Reply by iamronen** [reply]
this seems like a dead-end, can the issue be closed?

**Comment by alanna** [comment]
## Collective activity log #5195
We've done some good scoping and design work on this one that was prioritised last sprint already - I hope we can continue and implement soon. cc @Memo-Es

**Reply by Memo-Es** [reply]
The design will be ready this sprint; we talked about pushing it directly to implementation for the next one 
cc @znarf

**Reply by iamronen** [reply]
@Memo-Es can I reassign this issue to the Q3S1 board for pitching?
(it will disappear from the current Q2S2 board).

**Reply by Memo-Es** [reply]
Yeah @iamronen all good

**Comment by kewitz** [comment]
### https://github.com/opencollective/opencollective/issues/5435 Wise: Refactor expense payment to create Transactions only when transfer is concluded

Technical debt, we should make Wise expenses consistent with all other payout methods.

**Reply by iamronen** [reply]
added to product board for pitching

**Comment by BenJam** [comment]
Adding as a rollover as we're unlikely to manage it this last sprint:

## Updating OSC's eligibility criteria and onboarding process

Open Source Collective receives [5-6 applications a day, of which perhaps 1 is appropriate, with 1 a week is a great fit for us](https://opencollective-metabase.herokuapp.com/question/802-open-source-collective-application-frequency). Regardless of this we have to do the runaround making sure that the project actually meets the criteria that we care about:

- the licence
- the history of activity and the community
- the usage

The documentation for these changes are [in GH here](https://github.com/Open-Source-Collective/osc/pull/4/files?short_path=678defa#diff-678defa59b43eec1f38b7eb6e73c1ede157faa335fc702d3b05e024928dd6798) and in [GitBook here](https://app.gitbook.com/o/-LWSZizNMEjL8_DrMNdF/s/-MN_ynR0BjrarSoA9Z-A/~/changes/Z4Mvft9N2lqLtEBlxdvK/).

in the meantime, we automatically sign up a number of projects that aren't appropriate, because we use a proxy metric (number of stars) to judge whether a project should be accepted or not. This results in a lot of garbage (see above) projects being automatically accepted. 

Open Source Collective is going to update its eligibility criteria https://github.com/Open-Source-Collective/osc/issues/3  as a result, we need to update:

- [ ] the automatic approval process through GitHub
- [ ] the form for project not authenticated through GitHub (and open source adjacent projects) 

Specs and some research for both are in https://github.com/Open-Source-Collective/osc/pull/4

**Comment by Betree** [comment]
# https://github.com/opencollective/opencollective/issues/3438 Redesign the page to edit tiers

Been pushing for this one for two years now, I won't despair 🤪

- I think it has a great alignment with the [[Strategic Focus] Simplification #5486 ](https://github.com/opencollective/opencollective/issues/5486).
- Most collectives go through this flow that we're not proud of when they sign up.
_You never get a second chance to make a first impression._
- We identified this as a problem to implement further enhancements in multiple projects (https://github.com/opencollective/opencollective/issues/5368, https://github.com/opencollective/opencollective/issues/1798).

As mentioned in https://github.com/opencollective/opencollective/issues/5330#issuecomment-1091450737, I believe there is a way to simplify the implementation of this by relying on bounties. I actually have a PR started in https://github.com/opencollective/opencollective-frontend/pull/7403 that just deserves a bit of attention to properly kick-off the project.

Expected load on the engineering team:
- To create the skeleton: Medium (we probably want to do the listing part ourselves as the drag'n'drop feature deserves some refactoring)
- To create the specs: Simple
- To review the contributor's PRs: Medium. We can control the load by assigning the issues progressively

**Reply by Betree** [reply]
This would also help us move forward with technical debt, some components that we're mostly ready to kill are only used there. See https://github.com/opencollective/opencollective/issues/4830

**Reply by iamronen** [reply]
@BenJam do you have time to look at this and decide how to place it for prioritization?

**Reply by Memo-Es** [reply]
Big +1 to this one has recently come up in my calls about improving the contribution experience.

**Reply by BenJam** [reply]
The case for investing in tiers is strong from a use point of view:

![Screenshot 2022-07-06 at 11 29 17](https://user-images.githubusercontent.com/158833/177531884-7b99383e-43e9-4217-a2c4-69b74a02b8de.png)

@iamronen and I have been looking at this from the perspective of Open Source Collective helping collectives describe their ambitions and to link them to targets, tiers could be a part of that. If we don';t prioritise is this sprint i can see us putting OSC's weight behind it soon.

**Comment by Memo-Es** [comment]
## Contribution flow improvements (1st Round) **Issue pending**

We prioritized #4893 for this sprint as a research project to find out what was related to this issue for us to provide a more holistic solution and not jump to conclusions. 

The research process revealed that this issue is about improving the contribution experience in general certainly #4893 is one angle but not the only one, so I suggest considering this 'campaign' in a similar way as the #4829 and #5501, and dedicate some time next sprint to take a look collectively at this and shape it a bit more (Define concise projects and prioritize them)

**Comment by jfdh** [comment]
### Help Host Admin to mark expenses as incomplete faster #5713

As a Host Admin, I want to use the new "Mark as Incomplete" feature from the expenses list. I want to save time by not having to open the particular expense to mark it as incomplete. 

As we already have a preview of the attachment accessible from the expenses list, as well as the "Go to Pay" action available, it seems that the "Mark as Incomplete" action is the piece missing here.

**Strategic Priority:** Simplification
**High frequency - High impact**

**Comment by kaylarep** [comment]
## https://github.com/opencollective/opencollective/issues/5318 Sharing up-to-date press/media about e2c

If we don't do the airtable idea, at minimum, changing the order of the 'Resources' at the bottom.

to me, upon first glance, it just looks like the most recent article is from oct 18, 2021, not very recent.
at the very least, if we want to keep the oldest first, then i think the 'Resources' shouldnt be a carousel, but a list where you can view it all at once.

that's the only place on the whole page that shows any ongoing movement about e2c. without it, its a stagnant page of information, no idea of last update/if the conversation is even still going on.

**Comment by kaylarep** [comment]
## https://github.com/opencollective/opencollective/issues/5722 Edits to auto-emails for Virtual Cards

- hoping these shouldn't be too time-consuming. a couple things to discuss in here, but those conversations can happen at another time.

**Comment by BenJam** [comment]
## Notifications settings

Apologies for the delay here but, given what's been happening with the current sprint, I wanted to make sure that our request is accurate. Looking again at what was proposed last sprint https://github.com/opencollective/opencollective/discussions/5511#discussioncomment-2813811

> I propose that we make some quick fixes to email-based communication:
> - [ ] Allow admins to set a specific email address for contact requests #2263
> - [ ] [include email to host admins when contacting a pending collective](https://github.com/opencollective/opencollective/issues/4699#issuecomment-1041527950)

> and start to tackle #3357 by
> - [ ] introducing one overall setting (maybe essential, recommended, all)
    - ESSENTIAL (payment receipts, payouts, card payments, actions)
    - RECOMMENDED (confirmations of actions taken by others, platform drip onboarding) 
    - ALL (inc updates and conversations)*

AFIAK we are not going to complete the overall setting this sprint. Instead we've somehow implemented the more complex customisation options described later in the document. We I propose tackling these problems again. 

## Notifications content review

In addition, I think we should review the quality fo the emails that we send from a content, language and design point of view.

**Comment by iamronen** [comment]
This discussion is closed. 
Ideas for prioritization can now be submitted for Q3S2: #5741

**Reply by kaylarep** [reply]
@iamronen that link sends back to this same page

**Comment by iamronen** [comment]
thank you @kaylarep - corrected!
